### PR TITLE
Removes installer HTTP proxy options as they no longer exist.

### PR DESCRIPTION
**Pull request by jjeffers**

### CODE REVIEW

File: plugins/katello/3.15/advanced/http_proxies.md [hunk 1]
```diff
@@ -15,18 +15,6 @@ HTTP proxies can be created and then assigned to a product though bulk selection
 
 ## Creation
```
Comment (ekohl) [inline plugins/katello/3.15/advanced/http_proxies.md:16]: Should this header be dropped and the `Creating ...` header be decreased one level?

Comment (jjeffers) [reply plugins/katello/3.15/advanced/http_proxies.md:16]: I considered that, too. If no objections, lets try that.

**Comment by jjeffers** [comment]
Screenshot from 2020-05-05 07-38-15 @ekohl adjusting the header level.

**Review by ekohl — APPROVED** [review]
Looking at the screenshot, I'd consider dropping `HTTP Proxy Support` too since there's now a duplicate header. That was already present before you started so IMHO not a blocker.

**Pull request merged**